File: frontend/src/Models/Order.ts
export type Order = {
  Id: number;
  UserId: number;
  OrderCode: number;
  OrderDate: Date;
  Status: string;
  PaymentMethod: string;
  PaymentStatus: string;
  TotalAmount: number;
  ShippingAddress: string;
  ShippingFee: number;
  ShippingMethod: string;
  Note: string;
  CreatedDate: Date;
  UpdatedDate: Date;
  IsDeleted: boolean;
};

File: IoTMonitorApp.API/Models/Order.cs
using TypeGen.Core.TypeAnnotations;

namespace IoTMonitorApp.API.Models
{
    [ExportTsClass]
    public class Order : BaseEntity
    {
        public int Id { get; set; }
        public int UserId { get; set; }
        public string OrderCode { get; set; }
        public DateTime OrderDate { get; set; }
        public string Status { get; set; }
        public string PaymentMethod { get; set; }
        public string PaymentStatus { get; set; }
        public decimal TotalAmount { get; set; }
        public string ShippingAddress { get; set; }
        public decimal ShippingFee { get; set; }
        public string ShippingMethod { get; set; }
        public string Note { get; set; }

    }
}
